sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-workflowWrapUp.e2e-spec.ts

```diff
@@ -240,4 +250,531 @@ describe("AssessmentController(e2e)-workflowWrapUp", () => {
       assessImpactedApplicationReassessmentNeededMock,
     ).not.toHaveBeenCalled();
   });
+
+  it("Should populate the previousDateChangedReportedAssessment when the e-Cert was generated for the 'Original Assessment' and offering dates changed as a result of the reassessment.", async () => {
+    // Arrange
+    const savedApplication = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      undefined,
+      {
+        applicationStatus: ApplicationStatus.Completed,
+        currentAssessmentInitialValues: {
+          assessmentDate: now,
+          studentAssessmentStatus: StudentAssessmentStatus.Completed,
+        },
+        createSecondDisbursement: false,
+        firstDisbursementInitialValues: {
+          coeStatus: COEStatus.completed,
+          disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+        },
+      },
+    );
+    const originalAssessmentId = savedApplication.currentAssessment.id;
+    const differentStudyStartDate = new Date(
+      savedApplication.currentAssessment.offering.studyStartDate,
+    );
+    differentStudyStartDate.setDate(differentStudyStartDate.getDate() + 1);
```

Comment:
what `differentStudyStartDate.getDate() + 1` was supposed to do? Add one day, one sec, one millisec?

Use `addDays` from date-utils.ts please.

Reply:
Suggestion:
`const differentStudyStartDate = getISODateOnlyString(addDays(savedApplication.currentAssessment.offering.studyStartDate, 1));`